### PR TITLE
DiskPool Metrics Exporter - add committed size in bytes

### PR DESCRIPTION
**Pull request by tiagolobocastro**
chore: update controller dependency

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

feat(exporter/diskpool): add committed size in bytes

This is useful for thin provisioning to determine if a pool is overcommitted.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

File: exporter/src/pool/collector/pools_collector.rs [hunk 1]
```diff
@@ -112,6 +123,20 @@ impl Collector for PoolsCollector {
             let mut x = pool_used_size.collect();
             metric_family.extend(x.pop());
 
+            let pool_committed_size = match self
+                .pool_committed_size
+                .get_metric_with_label_values(&[node_name.clone().as_str(), p.name().as_str()])
+            {
+                Ok(pool_committed_size) => pool_committed_size,
+                Err(error) => {
+                    error!(%error,"Error while creating metrics(pool_used_size) with label values:");
```
Comment (niladrih) [inline exporter/src/pool/collector/pools_collector.rs:132]: There's a colon here, missing value?

Comment (tiagolobocastro) [reply exporter/src/pool/collector/pools_collector.rs]: Yes copy&paste from existing code, I fixed that the existing ones.

**Review by niladrih — APPROVED** [review]

**Review by sinhaashish — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**